### ISSUE TITLE
RegexException: failed to find pattern ""sts"\s*:\s*(\d+)

### ISSUE THREAD

**Issue by CWftw** [TeamNewPipe/NewPipeExtractor#174]
Hi,

I am getting this exception on this specific video: http://www.youtube.com/watch?v=cFDEGRc3PNA. I am using the latest commit from the dev branch. Similar issues seem to be from 2018. The video appears to be age restricted (should probably turn down your volume before watching)
```
org.schabi.newpipe.extractor.utils.Parser$RegexException: failed to find pattern ""sts"\s*:\s*(\d+)
```

**Comment by niuyaweiqq** [comment]
I also have this problem.

**Comment by niuyaweiqq** [comment]
http://www.youtube.com/watch?v=zQwOwaEElb8. Unable to download

**Comment by Stypox** [comment]
Will be fixed in #177
